src/diffusers/models/attention_processor.py

```diff
@@ -1022,6 +1089,7 @@ def __call__(self, attn: "Attention", hidden_states, encoder_hidden_states=None,
     AttnAddedKVProcessor2_0,
     LoRAAttnProcessor,
     LoRAXFormersAttnProcessor,
+    LoRAAttnAddedKVProcessor,
```

Comment:
Can we also add an xformers variant or [is this](https://github.com/huggingface/diffusers/blob/0ffac97933d8ca1487d4ae9c374504a0979f0c5d/src/diffusers/models/attention_processor.py#L162) still a problem?

Reply:
Hm, I haven't looked in a while. I think for this PR it would be best to just add the simplest attention processor we can

Reply:
It'd also maybe first focus on Torch 2.0 attention processor before xformers

Reply:
You Torch 2.0 implementation of LoRA? If so, yes, I am in agreement.